src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -808,7 +808,7 @@ UINT_PTR ExceptionTracker::FinishSecondPass(
     {
         CopyOSContext(pThread->m_OSContext, pContextRecord);
         SetIP(pThread->m_OSContext, (PCODE)uResumePC);
-#if defined(TARGET_UNIX) && !(defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_ARM))
+#if defined(TARGET_UNIX) && !(defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_ARM) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64))
         uAbortAddr = NULL;
 #else
```

Comment:
```suggestion
```
This should not be needed.

The only platform where the #if condition true is Linux x86 that looks like a bug.

Reply:
@alpencolt @clamp03 Is this still needed for Linux x86?

Reply:
> Is this still needed for Linux/Tizen x86?

I suppose yes, but not sure, because main branch has been crashing on linux x86 for a long time (all coreclr tests fail). We did not have time to investigate that yet.

Reply:
> > Is this still needed for Linux/Tizen x86?
> 
> I suppose yes, but not sure, because main branch has been crashing on linux x86 for a long time (all coreclr tests fail). We did not have time to investigate that yet.

@jkotas  OK, thank you.
If this still needed for Linux/Tizen x86 , perhaps we can modify it this way?
```suggestion
-#if defined(TARGET_UNIX) && !(defined(TARGET_AMD64) || defined(TARGET_ARM64) || defined(TARGET_ARM) || defined(TARGET_LOONGARCH64) || defined(TARGET_RISCV64))
+#if defined(TARGET_UNIX) && defined(TARGET_X86)
         uAbortAddr = NULL;
 #else
         uAbortAddr = (UINT_PTR)COMPlusCheckForAbort(uResumePC);
```